### DISCUSSION TITLE
Suppressing ping replies

### DISCUSSION THREAD

**Discussion by 03302024** [tnodir/fort#371] [Q&A] [answered]
Testing for open Common Ports using [ShieldsUP!](https://www.grc.com/x/ne.dll?bh0bkyd2), I get this report:

----------------------------------------------------------------------

GRC Port Authority Report created on UTC: 0000-00-00 at 00:00:00

Results from scan of ports: 0, 21-23, 25, 79, 80, 110, 113, 
                            119, 135, 139, 143, 389, 443, 445, 
                            1002, 1024-1030, 1720, 5000

    0 Ports Open
    0 Ports Closed
   26 Ports Stealth
---------------------
   26 Ports Tested

ALL PORTS tested were found to be: STEALTH.

TruStealth: FAILED - ALL tested ports were STEALTH,
                   - NO unsolicited packets were received,
                   - A PING REPLY (ICMP Echo) WAS RECEIVED.

----------------------------------------------------------------------

It is all good except it says a ping reply is received. I am on a WiFi router. Can this affect that? What is the proper Fort setting to suppress ping replies?

**Comment by tnodir** [comment, marked as answer]
Create Blocking "Global Rule, applied before App Rules":
```
dir(IN):proto(ICMP):icmp_type(8)
```

![global-block-ping](https://github.com/user-attachments/assets/20f5ac68-4102-487b-91d5-8dcb7ed4b9e4)
![global-block-ping-rule](https://github.com/user-attachments/assets/ab918f49-219d-4e55-a01e-8216148781da)

**Comment by 03302024** [comment]
Thank you for your reply. I did that but still I get that "A PING REPLY (ICMP Echo) WAS RECEIVED" message. I am wondering who is actually replying to the ping. Is it the router itself or one of the many computers that are connected to the same router.

Also, the solution that you are describing above, could be added as a checkbox to the Options window.

**Reply by tnodir** [reply]
It should be a router itself. Check the router settings for ICMP requests.

**Reply by 03302024** [reply]
OK, thanks.

**Comment by 03302024** [comment]
Just wanted to say that with the new 3.15.8 version, I get this report:

GRC Port Authority Report created on UTC:  at 

Results from scan of ports: 0, 21-23, 25, 79, 80, 110, 113, 
                            119, 135, 139, 143, 389, 443, 445, 
                            1002, 1024-1030, 1720, 5000

    0 Ports Open
    0 Ports Closed
   26 Ports Stealth
---------------------
   26 Ports Tested

ALL PORTS tested were found to be: STEALTH.

TruStealth: PASSED - ALL tested ports were STEALTH,
                   - NO unsolicited packets were received,
                   - NO Ping reply (ICMP Echo) was received.


It says no ping replied, which is good. However, option "Stealth mode (Prevent port scanning)" is unchecked.

**Reply by tnodir** [reply]
1. Windows Firewall installs its own Stealth mode filters by default. You have to [disable those filters first](https://www.google.com/url?sa=t&source=web&cd=&ved=2ahUKEwi-gciopJuLAxU58bsIHUkaJhEQFnoECEkQBQ&url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Ftroubleshoot%2Fwindows-server%2Fnetworking%2Fdisable-stealth-mode&usg=AOvVaw37MHtVj1uJS5r_9Hm3vpX8&opi=89978449).
2. See the #396 how to test the Stealth mode.

**Reply by 03302024** [reply]
Well, that information is way above my head. I will set option "Stealth mode (Prevent port scanning)" checked, and I am happy with the "no ping reply" report.
